tests/waitForEvent.js

```diff
@@ -13,9 +13,9 @@ export function waitForEvent(event, subject = document) {
   const remove = isVue ? '$off' : 'removeEventListener';
 
   return new Promise((resolve) => {
-    const listener = () => {
+    const listener = (e) => {
       subject[remove](event, listener);
```

Comment:
Gaat dit nog goed? aangezien je de andere `e` noemt. Ik neem aan dat dit ook `event` is.